react/week3/src/components/Todo.js

```diff
@@ -0,0 +1,63 @@
+import React , { useState } from 'react'
+
+function Todo({ todo, index, completeTodo, removeTodo, updateTodo }) {
+ 
+  const [editTask, seteditTask] = useState(false);
+  const [state, setState] = useState({newText: todo.text, newDeadline:todo.deadline});
+
+    return (

```

Comment:
This part in the return statement is unreadable.
Only use ternary statements when you very little code and only using 1 or 2 lines .
It doesn't help that you break it up by doing this
      ) : (
on a new line.
Try to refactor